### ISSUE TITLE
Relative component - invalid attribute name

### ISSUE THREAD

**Issue by mukimov** [rebassjs/rebass#482]
Just upgraded to 2.3.2 and seems that `Relative` component is broken, it works but prints warnings.
Below are props that passing to component, don't know form where this props are coming from: 
```
{
  "0": "s",
  "1": "p",
  "2": "a",
  "3": "c",
  "4": "e"
}
```

**Comment by jxnblk** [comment]
Thanks! (that is a very cool bug)

**Comment by jxnblk** [comment]
Should be fixed in v2.3.3
